std/stdio.d

```diff
@@ -1522,6 +1524,139 @@ for every line.
     }
 
 /**
+Creates a file with a randomly generated name and returns an open File to it.
+
+There is nothing special about the file created other than the fact that its
+name is randomly generated, and it defaults to being in the temp directory on
+the system. It will only be deleted if explicitly deleted or if it's in the
+temp directory, and the temp directory is cleared out (as happens on system
+startup or shutdown on some systems).
+
+The file is created with R/W permissions and opened with $(D "w+b"). On POSIX
+systems, the permissions are restricted to the current user, though the
+effective permissions are modified by the process' umask in the usual way.
+
+Params:
+    prefix = Prefix for the generated file name.
+    suffix = Suffix for the generated file name (which also provides a way to
+             give the file an extension).
+    dir    = Directory of the temporary file. Defaults to the result of
+             $(XREF file, tempDir).
+  */
+    static File tempFile(string prefix = null, string suffix = null)
```

Comment:
If the documentation is generated with warnings it will complain about no parameter for `dir`:

```
Warning: Ddoc: function declaration has no parameter 'dir'
```

If you put the documentation on the other overload instead it will not complain.